packages/jellyfish-transaction-builder/src/txn/txn_builder.ts

```diff
@@ -15,7 +15,7 @@ import { calculateFeeP2WPKH } from './txn_fee'
 import { TxnBuilderError, TxnBuilderErrorType } from './txn_builder_error'
 import { EllipticPair } from '@defichain/jellyfish-crypto'
 import { Network } from '@defichain/jellyfish-network'
-import { ListUnspentQueryOptions } from '@defichain/jellyfish-api-core/dist/category/wallet'
+import { ListUnspentQueryOptions } from '@defichain/jellyfish-api-core/src/category/wallet'
```

Comment:
```suggestion
import { ListUnspentQueryOptions } from '@defichain/jellyfish-api-core'
```

It should be this, but frankly speaking jellyfish-transaction-builder shouldn't have access to jellyfish-api-core since it's agnostic

Reply:
fixed https://github.com/BirthdayResearch/jellyfishsdk/pull/2158/commits/32a416bff1a061c95529ff1ca290749a8d2ce493
@pierregee